stdlib/3/subprocess.pyi

```diff
@@ -250,7 +250,12 @@ STDOUT = ...  # type: int
 if sys.version_info >= (3, 3):
     DEVNULL = ...  # type: int
     class SubprocessError(Exception): ...
-    class TimeoutExpired(SubprocessError): ...
+    class TimeoutExpired(SubprocessError):
+        cmd = ...  # type: List[str]
+        timeout = ...  # type: int
+        output = ...  # type: Optional[str]
```

Comment:
Your example shows these to be `bytes` actually.

Reply:
Presumably this depends on whether an `encoding=` argument is passed to subprocess.run. We should probably just make it Any.

Reply:
Good catch. These appear to always be bytes.